//chp5q1

try {
  let a = heloo;
} catch (e) {
  //used to handle erros
  console.log(e.name);
  console.log(e.message);
}
